### ISSUE TITLE
feature extracted from matlab does not match that from caffe

### ISSUE THREAD

**Issue by ghost** [BVLC/caffe#524]
Hi,
   The model says about loss = 0.05 at the training time.
   But when I use this model ,the same train data for MATLAB wrapper, and get the softmax output, the loss ~= 0.4 . Which does not match the train loss.
    The MATLAB loss function is : loss = -log(prob);
     I'm sure that the prototxt is OK.
